### PR TITLE
add Korean HateSpeech dataset

### PR DESCRIPTION
**Pull request by stevhliu**

### CODE REVIEW

**Comment by cceyda** [comment]
It looks like you forgot to `make style` (I forget it a lot too 🤦 )
+ add dummy data

**Comment by stevhliu** [comment]
hi @cceyda 👋, thanks for the hint! it looks like i've run into some other errors though in `_split_generators` or `_generate_examples`.  do you have any idea of what's wrong here? 😅

**Comment by cceyda** [comment]
I get the same errors on another pr too, so it probably has something to do with circleci, waiting on help.

**Comment by lhoestq** [comment]
the `RemoteDatasetTest ` error on the CI is fixed on master so it's fine

File: datasets/kor_hate/README.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+---
+YAML tags:
```
Comment (lhoestq) [inline datasets/kor_hate/README.md:2]:
```suggestion
```

File: datasets/kor_hate/README.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+This dataset is curated by Jihyung Moon, Won Ik Cho and Junbum Lee.
+
+### Licensing Information
+
+[N/A]
+
+### Citation Information
+
+```python
```
Comment (lhoestq) [inline datasets/kor_hate/README.md:155]:
````suggestion
```
````

**Review by lhoestq — APPROVED** [review]
Thanks for adding this one !

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**